roles/testnode/tasks/pip.yml

```diff
@@ -1,16 +1,18 @@
 ---
+# python-pip is installed from epel repo which doesn't exist on fedora.
+# package is installed during kickstart instead.
```

Comment:
OK.  So it's not part of kickstart; it's part of the post-install ansible.  So how about "package is installed during post-install ansible (or name the task specifically, or both)